TSQL_Parser/TSQL_Parser/Clauses/Parsers/TSQLSubqueryHelper.cs

```diff
@@ -58,6 +58,12 @@ internal static class TSQLSubqueryHelper
 							nestedLevel++;
 							expression.Tokens.Add(tokenizer.Current);
 						}
+						else if (tokenizer.Current.IsKeyword(TSQLKeywords.VALUES))
```

Comment:
What caused you to need to parse VALUES specifically right here? I would have expected it to work as is, and since we're not returning back the complex objects, the special handling doesn't seem beneficial.